x-pack/plugins/secops/server/graphql/overview/schema.gql.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import gql from 'graphql-tag';
+
+export const siemOverviewSchema = gql`
+  type SiemOverviewNetworkData {
```

Comment:
This schema is pretty made up. I'm not sure what is and is not required (!)

Reply:
There's been some discussion around required fields lately and the guideline as been to not mark scalar fields  as required if it is necessary for the user to know that there is an absence of data. Metric fields are kind of up in the air, as there is benefit in knowing if no value was returned which points to a missing or mis-configured index, but we also have the `WithSource` component which checks for index configuration before even querying. I think for now you'd be fine marking them as required and supplying a sensible default (`0`) in the `elasticsearch_adapter` before sending back to the client. We can revisit once we get a better sense of customer configurations and what UX is determined to be the best experience (i.e. seeing `--` for all kinds of metrics vs `0`'s and whether or not we provide additional information on how the user can resolve the `--`'s). Hope that helps! :)